examples/sglang/utils/sgl_http_server.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import uvicorn
+from fastapi import FastAPI, HTTPException
+import sglang as sgl
+import asyncio
+from sglang.server_args import ServerArgs
+from dynamo.runtime import DistributedRuntime
+
+class SglangHttpServer:
+    # TODO: add decode client conditional as well
+    def __init__(self, engine: sgl.Engine, server_args: ServerArgs, drt: DistributedRuntime, port: int = 9001):
+        self.engine = engine
+        self.port = port
+        self.app = FastAPI()
+        if server_args.disaggregation_mode != "null":
+            self.decode_flush_client = drt.namespace("dynamo").component("decode").endpoint("flush_cache").client()
+
+        self.setup_routes()
+        
+    def setup_routes(self):
+        @self.app.post("/flush_cache")
+        async def flush_cache():
+            """Flush the radix cache."""
+            try:
+                # this targets the prefill/aggregated worker
+                asyncio.create_task(self.engine.tokenizer_manager.flush_cache())
+                
+                # then we target the decode worker over nats
+                await self.decode_flush_client.flush_cache()
```

Comment:
Is expectation that user has to curl /flush_cache N times for number of workers? Or that this http server implementation should loop over all instances for the user?